SetpointTag:                0

MapTopic:                   '/voxel_map'

PoseTopic:                  '/traj_start_trigger'

CtrlTopic:                  '/traj_follow_start_trigger'

CmdTopic:                   '/drone/planning/traj_follow'

TargetTopic:                '/move_base_simple/goal'

DilateRadius:               0.5

VoxelWidth:                 0.25

MapBound:                   [-75.0, 25.0, -25.0, 25.0, 0.0, 5.0]

TimeoutRRT:                 0.02

MaxVelMag:                  4.0  ##4.0

MaxBdrMag:                  1.5 ##2.1

MaxTiltAngle:               0.26 ##1.05

MinThrust:                  2.0

MaxThrust:                  10.0 ##12.0

VehicleMass:                4.0

GravAcc:                    9.8

HorizDrag:                  0.70

VertDrag:                   0.80

ParasDrag:                  0.01

SpeedEps:                   0.0001

WeightT:                    20.0

ChiVec:                     [1.0e+4, 1.0e+4, 1.0e+4, 1.0e+4, 1.0e+5]

SmoothingEps:               1.0e-2

IntegralIntervs:            16

RelCostTol:                 1.0e-5

X_Wide:                     2.0

Y_Wide:                     2.0

Z_Wide:                     1.0

Z_min:                      0.5